perllib/FixMyStreet/App/View/Web.pm

```diff
@@ -98,6 +99,24 @@ sub _space_slash {
     return $t;
 }
 
+=head2 markup_factory
+
+This returns a function that will allow updates to have markdown-style italics.
+
+=cut
+
+sub markup_factory {
+    my ($c, @args) = @_;
+    my $update = $c->stash()->get('update');
+    my $user = $update ? $update->user : undef;
+    return sub {
+        my $text = shift;
+        return $text unless $user && ($user->from_body || $user->is_superuser);
```

Comment:
This (and the `$update` stuff above) is all quite specific to the changing category feature. Would it be better to handle this logic in the template instead? I'm just a bit worried that this new attractive looking `markup` filter might cause confusion because it won't actually work everywhere you might expect it to.